### ISSUE TITLE
Patch for /slides.html5rocks.com/js/utils.js

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#548]
```
Prevents javascript error when user chooses not to use a TOC slide.
```

Original issue reported on code.google.com by `sscaff...@gmail.com` on 25 Jun 2011 at 8:36

Attachments:
- [utils.js.patch](https://storage.googleapis.com/google-code-attachments/html5rocks/issue-548/comment-0/utils.js.patch)

**Comment by GoogleCodeExporter** [comment]
```
I was building a set of slides using the demo as a base and found that when you 
don't have a TOC slide, the browser gets stuck showing the Loading... message.  
This happens because an exception is thrown when trying to insert the HTML into 
the TOC element when it doesn't exist :). This patch fixes that. 
```

Original comment by `sscaff...@gmail.com` on 25 Jun 2011 at 8:38

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbidelman@chromium.org` on 8 Sep 2011 at 11:58
- Changed state: **as**
- Added labels: **Priority-P2**, **Milestone-7.2**

**Comment by GoogleCodeExporter** [comment]
Original comment by `han...@google.com` on 9 Sep 2011 at 2:07
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Revision a0f9a91cf479.
```

Original comment by `han...@google.com` on 22 Nov 2011 at 7:30
- Changed state: **Fixed**
